templates/reporting/report_by_user.html.twig

```diff
@@ -19,11 +19,24 @@
                 {{ widgets.username(user) }}
             {% endif %}
             {{ form_widget(form.date) }}
+            <div class="btn-group">
+                <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    <i class="{{ 'filter'|icon }}"></i> <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu checkbox-menu">
+                    <li>
+                        {{ form_widget(form.sumType) }}
+                    </li>
+                </ul>
+            </div>
         {% endblock %}
         {% block box_body_class %}{{ box_id }} table-responsive no-padding{% endblock %}
         {% block box_body %}
             {% set totals = {'totals': 0} %}
             {% set columns = 2 %}
+            {% set fieldName = form.sumType.vars.value %}
```

Comment:
Not sure if this is the correct way to access this data. It's either 'duration', 'rate' or 'internalRate'.